main.md

```diff
@@ -379,38 +379,32 @@ are included for formatting and readability.
 !---
 Figure: Example JWT content of a `DPoP` proof {#dpop-proof}
 
-Of the HTTP content in the request, only the HTTP method and URI are
-included in the DPoP JWT, and therefore only these 2 headers of the request
-are covered by the DPoP proof and its signature.
-The idea is sign just enough of the HTTP data to
-provide reasonable proof-of-possession with respect to the HTTP request. But 
-that it be a minimal subset of the HTTP data so as to avoid the substantial 
-difficulties inherent in attempting to normalize HTTP messages. 
-Nonetheless, DPoP proofs can be extended to contain other information of the
-HTTP request (see also (#request_integrity)).
+DPOP JWT conveys only the HTTP method and the request target.
+This is to provide a reasonable correlation with the HTTP request. 
+Nonetheless, DPoP proofs can be extended to contain other information
+related to the HTTP request (see also (#request_integrity)).
```

Comment:
I disagree with the deletion of the text above about the two headers of the request, etc.  The replacement text does not equivalently convey the information that would have been deleted.

Reply:
Yes, please restore the original paragraph of text.

Reply:
Pls see:

- https://github.com/danielfett/draft-dpop/pull/146/files#r855935610
- https://github.com/danielfett/draft-dpop/pull/146/files#r850816046